recipe/meta.yaml

```diff
@@ -142,19 +232,20 @@ outputs:
     version: {{ version }}
     script: build-pyomniscidb.sh
     build:
-      skip: true  # [cuda_compiler_version in ("10.2",)]
+      skip: true  # [cuda_compiler_version in ("11.0",)]
       string: {{ py_build_string }}
     requirements:
       host:
         - python
         - setuptools_scm
         - packaging >=20.0
         - rbc >=0.2.2
+        - pytest
```

Comment:
why do we require `pytest` in `host`? I thought conda had ways of specifying test-only requirements, and I see you're using it down below

Reply:
Indeed, for pyomniscidb, pytest in host is not required. Removed.